config/openhpc.yml

```diff
@@ -101,3 +101,21 @@ openhpc_packages:
   - infiniband-diags
   - emacs
 #  - git-lfs
+
+users:
+- username: centos
+  groups: ['wheel', 'systemd-journal']
+  ssh_key:
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDCLva2hRoqcA77XoK0t7aPmip4CmxM6SdsnJdbN4i0l72Rf6HMNH2ZR5p+eZPxaxiJgmtQ5MpquNt775lFSGVIV7oyAKDdYuLA/8KXoKje4EsCMfA9m6Eg8urjrPprelTAl7xsem1eH9c/Jk6TFbJSyMrY7kvgNo1KZ+D24F5cCSSRncy0s5GoPWEJfdnSHdjvxc97vhSPY9KwsaMdbpj+u+1WlUDeoLjMsWLPDsqnIHLwYN74f9jfTvsoQxw9Wq3dmzRvy8sQkJ1OoAeuuM4B4MKel7U6QqrhMYyIUG2pcSBBICtT2Q5V4RhnBIqQrL8w1efq+TeD/ETSuQgFBiq1 sclt100@ilab-gate.ilab.cluster"
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCoHB+D6iI859Fq4Wg3iWDPf+miX+nXDx8Y8F20C5wMa7uHFYWWp0gfhMslI3ntWvlPf9J7C8AQz/kzPeuvM9oEZUkBuH6oYtYrJ9nA6+IJqT/xbQGUy56hkVVFH8Etff9CzseczzhLPOd/CEj4irLdi4aoj/5V8tqvdAbnOGwixPRPmVKVvEVpFiAlvRi+0MCaWHF2lF4BVWJsRKKGf79DFOrhfI9+BefuD9essaCOuFfhkHSxngBIjicYJKSedMFoWaOSY35820P/6RnmbKjH/mMQ0yuEr7SgLZ3gdTN2crrDj0q8+yz5MHfuxPdtj9dOYb8yP8MahbOl91qIV5BL John@Lenovo-PC"
+  - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDT7FfKdhVmT/HturOK6sLRKcqFYDnfbKWVE8Lv+KntJCBN0pGk5PECCmmA7/5JzAzfba4nmMpI20byJ0VCphzTbYFHtsZNd6Ci4NIKb5GrxP1f9RN3BYlmm1rJ5OgoXjvg0VpUiUnkLsVYkHfzUTx6tDHj8zV2DQYWGg2BtZk8YSEnigK6ppZcBuRwhY1wmKEodeXus6vik20v+PGDAlSmSlhHsYUOe8/GNhs6fZ9a9XEVHw/pPxYa3HmNGgUiJYOxSA0DLq8HzxQDoSgpMMeG2bD36DsHSnnbLNUtH3+DmveX2FxWWHRKG4IgW5O0UdLcGzrakZ6OFi/bUL0bBge7 bharat@dev-director.dev-cluster.hpc.cam.ac.uk"
+
+# Swarm SIP: manila mount
+os_manila_mount_os_config_name: alaska
```

Comment:
I am not sure this will work because the localhost does not have /etc/openstack/clouds.yaml installed.  Look at group_vars/openstack for credentials drawn from the user environment.